.github/workflows/main.yml

```diff
@@ -10,10 +10,21 @@ jobs:
   spellcheck:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Spellcheck Action
-        uses: rojopolis/spellcheck-github-actions@0.2.0
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools
```

Comment:
I'm gonna ragequit when I find out I have to mess around with python versions and pip to write simple markdown... :P